### PR TITLE
Added Responding to violations topic

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Based on 
- https://help.stackrox.com/docs/respond-to-violations/
- https://help.stackrox.com/docs/respond-to-violations/view-violation-details/

Preview at https://openshift-docs-git-violations-res-gnelson.vercel.app/openshift-acs/master/operating/respond-to-violations.html

### CODE REVIEW

File: operating/respond-to-violations.adoc [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Using {product-title} you can view policy violations, drill down to the actual cause of the violation, and take corrective actions.
+
+{product-title} built-in policies identify a variety of security findings, including vulnerabilities (CVEs), violations of DevOps best practices, high-risk build and deployment practices, and suspicious runtime behaviors.
+Whether you use the default out-of-box security policies or use your own custom policies, {product-title} reports a violation when an enabled policy fails.
+
+include::modules/violations-view.adoc[leveloffset=+1]
+
+[id="view-violation-details"]
+== Viewing violation details
+When you select a violation in the *Violations* view, the *Violation Details* open in a panel on the right.
```
Comment (bergerhoffer) [inline operating/respond-to-violations.adoc:18]: the way this is worded sounds odd with *Violation Details* being styled as the name of the panel. I expected to see "page" or "panel" after it, with the "the" in front of it. Suggest rewording a little, maybe:

```
the *Violation Details* panel opens on the right.
```
or
```
the violation details open in a panel on the right.
```

File: operating/respond-to-violations.adoc [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+The *Violation Details* panel shows detailed information grouped by multiple tabs.
+
+include::modules/violation-view-violation-tab.adoc[leveloffset=+2]
+
+[id="using-comments-and-tags"]
+=== Using comments and tags
+You can use tags and comments to specify what is happening with violations to keep your team up to date.
+Comments allow you to add text notes to violations and tags allow you to categorize your violations.
+
+:smallporv: violation
```
Comment (bergerhoffer) [inline operating/respond-to-violations.adoc:29]: What are these? I don't see these attributes used anywhere

File: operating/respond-to-violations.adoc [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+[id="view-violation-details"]
+== Viewing violation details
+When you select a violation in the *Violations* view, the *Violation Details* open in a panel on the right.
+
+The *Violation Details* panel shows detailed information grouped by multiple tabs.
+
+include::modules/violation-view-violation-tab.adoc[leveloffset=+2]
+
+[id="using-comments-and-tags"]
+=== Using comments and tags
```
Comment (bergerhoffer) [inline operating/respond-to-violations.adoc:25]: Just checking that this is the right spot for this. Looking at the TOC, it seemed out of place with all of the other sections being a particular "tab". Is this particular to the violations tab and should be indented?

File: modules/violation-view-policy-tab.adoc [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+//
+// * operating/respond-to-violations.adoc
+:_module-type: CONCEPT
+[id="violation-view-policy-tab_{context}"]
+= Policy tab
+
+[role="_abstract"]
+The *Policy* tab of the *Details* panel displays details of the policy which caused the violation.
+
+[discreet]
```
[the remark this answers is not in the document]
Comment (gaurav-nelson) [reply modules/violation-view-policy-tab.adoc:11]: 🤦🏼‍♂️

File: operating/respond-to-violations.adoc (hunk 2 above)
Comment (gaurav-nelson) [reply operating/respond-to-violations.adoc:29]: The changes for those attributes are part of https://github.com/openshift/openshift-docs/pull/33962
- PS [my comment](https://github.com/openshift/openshift-docs/pull/33962#pullrequestreview-692470610)

Comment (bergerhoffer) [reply operating/respond-to-violations.adoc:29]: Cool, I'll add a comment there, but this is fine then. Thanks!

**Review by bergerhoffer — COMMENTED** [review]
LGTM

**Pull request merged**